templates/front/my_failed_checks.html

```diff
@@ -0,0 +1,277 @@
+{% extends "base.html" %}
```

Comment:
The contents of this file look exactly like the ones in checks. Think of re-using this the checks template to instead of duplication this code.